README.md

```diff
@@ -492,6 +494,12 @@ Whether to automatically mount the `buildkite-agent` binary and associated envir
 
 Default: `false`
 
+### `mount-checkout` (optional, boolean)
+
+Whether to automatically mount the current working directory which contains your checked out codebase. Mounts onto `/workdir`, unless `workdir` is set, in which case that will be used.
+
+Default: `true`
```

Comment:
I would definitely set it to `false` as default. I don't think there is a need to make this change backwards-incompatible when it would be the same amount of effort with no risk to make it so.